tests/antares/services/api_services/test_study_api.py

```diff
@@ -52,6 +53,184 @@ class TestCreateAPI:
         ServiceFactory(api, study_id).create_thermal_service(),
         ServiceFactory(api, study_id).create_renewable_service(),
     )
+    json_output = [
```

Comment:
Wasn't sure about the json for the outputs in the unit tests. Since it's very large, should I just include the relevant keys or leave it like that

Reply:
Indeed we don't need all this. You can just simplify by:
``` python
{
    "name": "20241217-1115eco-sdqsd",
    "type": "economy",
    "settings": {},
    "archived": False,
}
```
for instance